{
  "Header": {
    "Original": "CVE-2024-25178 (LuaJIT through 2.1 has an out-of-bounds read in the stack-overflow han ...)",
    "ID": "CVE-2024-25178",
    "Description": "(LuaJIT through 2.1 has an out-of-bounds read in the stack-overflow han ...)"
  },
  "Annotations": [
    {
      "Original": "- luajit 2.1.0+openresty20240314-1",
      "Type": "package",
      "Package": "luajit",
      "Kind": "fixed",
      "Version": "2.1.0+openresty20240314-1"
    },
    {
      "Original": "NOTE: https://github.com/LuaJIT/LuaJIT/issues/1152",
      "Type": "NOTE",
      "Description": "https://github.com/LuaJIT/LuaJIT/issues/1152"
    },
    {
      "Original": "NOTE: Fixed by: https://github.com/LuaJIT/LuaJIT/commit/defe61a56751a0db5f00ff3ab7b8f45436ba74c8 (v2.1)",
      "Type": "NOTE",
      "Description": "Fixed by: https://github.com/LuaJIT/LuaJIT/commit/defe61a56751a0db5f00ff3ab7b8f45436ba74c8 (v2.1)"
    }
  ]
}